OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -251,6 +251,21 @@ void LoadMapIntoEditor(string uid)
 				() => { Game.CloseServer(); SwitchMenu(MenuType.MapEditor); });
 		}
 
+		string CensusQuery()
+		{
+			if (!Game.Settings.Game.PlayerCensusParticipation)
+				return null;
+
+			return "?id={0}&platform={1}&os={2}&runtime={3}&lang={4}&version={5}&mod={6}".F(
+				Uri.EscapeUriString(Game.Settings.Game.UUID),
+				Uri.EscapeUriString(Platform.CurrentPlatform.ToString()),
+				Uri.EscapeUriString(Environment.OSVersion.ToString()),
+				Uri.EscapeUriString(Platform.RuntimeVersion),
+				Uri.EscapeUriString(System.Globalization.CultureInfo.InstalledUICulture.TwoLetterISOLanguageName),
+				Uri.EscapeUriString(ModMetadata.AllMods["all"].Version),
```

Comment:
Can we log the current mod's ID and version instead?  IMO this is more useful than the "all" version and the mod title.